moped-database/migrations/1657564405501_alter_table_public_moped_project_add_column_interim_project_id/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."moped_project" add column "interim_project_id" integer
+ null;
```

Comment:
No problem here, but I wonder why there is a linefeed chopping off one keyword from this `alter` statement. Works fine, just curious. Maybe that's right at 80 characters. Same in the other migration.